projects/packages/forms/src/blocks/form-progress-indicator/editor.scss

```diff
@@ -0,0 +1,35 @@
+.jetpack-form-progress-indicator {
+	--jp-form-progress-height: 0.75rem;
+	
+	// Use theme colors with fallbacks
+	--jp-form-progress-bg-color: #e0e0e0;
+	--jp-form-progress-color: #1e1e1e;
+
+	// Essential structure
+	width: 100%;
+	height: var( --jp-form-progress-height );
+	position: relative;
+	overflow: hidden;
+	background-color: var( --wp--preset--color--secondary, var( --wp--preset--color--background, var( --jp-form-progress-bg-color ) ) );
+	border-radius: calc( var( --jp-form-progress-height ) / 2 );
+
+	// Progress bar element - rendered as a child element to match the frontend
+	.jetpack-form-progress-indicator-bar  {
+		content: '';
+		display: block;
+		position: absolute;
+		top: 0;
+		left: 0;
+		height: 100%;
+		width: 1%;
+		min-width: var( --jp-form-progress-height );
+		transition: width 0.3s ease-in-out;
```

Comment:
Could be wrapped in `@media not (prefers-reduced-motion) { }` to avoid showing animations to those with "prefers reduced motion" setting in their browser.